create-k3d-cluster

```diff
@@ -10,11 +10,17 @@ API_PORT=8555
 EXPOSE=8080:80@loadbalancer
 VOLUME=$BASEDIR/.volume
 
-mkdir -p .volume
+mkdir -p $VOLUME
+if [ -f /etc/machine-id ]; then
+  cp -f /etc/machine-id $VOLUME/
+else
+  touch $VOLUME/machine-id
+fi
```

Comment:
Should this be explained with a comment? It might not be obvious to readers why this is happening.

Reply:
Good call :+1:

Reply:
Fixed in c5d37f7dae886fb8b19c09218b7b7e607fba1224